### PR TITLE
docs: created new documentation for Velero Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

File: docs/main/Solutioning/backup-and-restore/velero-br-fs.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+When CSI snapshots are enabled, Velero leverages the Kubernetes CSI driver to create snapshots of persistent volumes. These snapshots can be used to restore data to the same or different Kubernetes clusters.
+
+## Details of Setup
+
+### Install Velero with CSI Snapshot Support Enabled
+
+Install Velero (V1.13.2) with the `velero-plugin-for-csi` and with any S3 object storage. In this example, we are using the Google cloud bucket as our BackupStorageLocation. Refer to [Velero Documentation](https://velero.io/plugins/) for instructions on setting up the BackupStorageLocation.
+
+As an example we will be using Google Cloud Platform (GCP) and OpenEBS Plugin.
```
Comment (datacore-tilangovan) [inline docs/main/Solutioning/backup-and-restore/velero-br-fs.md:37]: We can remove the "plugin" here. We are not using plugin here. we have removed this from command must be a typo here.

As an example we will be using Google Cloud Platform (GCP) and OpenEBS

File: docs/main/Solutioning/backup-and-restore/velero-br-fs.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+velero backup create my-fs-backup --snapshot-volumes --include-namespaces=mayastor-app --volume-snapshot-locations=default --storage-location=default --snapshot-move-data
+```
+
+**Output**
+
+```
+Backup request "my-fs-backup" submitted successfully.
+```
+
+Run `velero backup describe my-fs-backup` or `velero backup logs my-fs-backup` for more details.
```
Comment (datacore-tilangovan) [inline docs/main/Solutioning/backup-and-restore/velero-br-fs.md:171]: This is also part of output

File: docs/main/Solutioning/backup-and-restore/velero-br-fs.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+velero restore create my-fs-restore --from-backup my-fs-backup --restore-volumes=true --namespace-mappings mayastor-app:mayastor-app-restore
+```
+
+**Output**
+
+```
+Restore request "my-fs-restore" submitted successfully.
+```
+
+Run `velero restore describe my-fs-restore` or `velero restore logs my-fs-restore` for more details.
```
Comment (datacore-tilangovan) [inline docs/main/Solutioning/backup-and-restore/velero-br-fs.md:234]: same here, this is also part of the output. when we submit the backup or restore, this logs and describe command pops up as part of the request.

File: docs/main/Solutioning/backup-and-restore/velero-br-fs.md [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+parameters:
+  protocol: nvmf
+  repl: "3"
+  thin: "true"
+provisioner: io.openebs.csi-mayastor
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+```
+
+In this example, We have deployed a sample Nginx test application for backup and restore
```
Comment (avishnu) [inline docs/main/Solutioning/backup-and-restore/velero-br-fs.md:129]: Should we mention something to this effect at the top?
Something like "In this guide, using Velero we will take a backup of a sample Nginx application with a Mayastor PV from a cluster,  transfer the backup to an object store and restore on a different cluster"

Comment (avishnu) [inline docs/main/Solutioning/backup-and-restore/velero-br-fs.md:129]: Also the volume mode as file-system needs to be mentioned here.

**Review by datacore-tilangovan — APPROVED** [review]

File: docs/main/Solutioning/backup-and-restore/velero-br-fs.md [hunk 5]
```diff
@@ -14,13 +14,15 @@ Using Velero for backup and restore operations with Replicated PV Mayastor snaps
 
 Velero can support any volume provider that has a Container Storage Interface (CSI) driver with snapshotting capability.
 
+In this guide, we will utilize Velero to capture a backup of a sample Nginx application with a Replicated PV Mayastor from a cluster, transfer the backup to an object store, and restore it on a different cluster.
```
Comment (avishnu) [inline docs/main/Solutioning/backup-and-restore/velero-br-fs.md:17]: create instead of capture?

**Review by avishnu — APPROVED** [review]

**Pull request merged**